specification/inference/put_deepseek/request/PutDeepSeekRequestExample1.yaml

```diff
@@ -0,0 +1,80 @@
+summary: A completion task
+description: Run `PUT _inference/completion/deepseek-completion` to create an inference endpoint that performs a completion task.
+method_request: 'PUT _inference/completion/deepseek-completion'
+# type: "request"
+value: |-
+  {
+      "service": "deepseek",
+      "service_settings": {
+          "api_key": "DeepSeek-Api-key",
+          "model_id": "deepseek-chat"
+      }
+  }
+alternatives:
+  - language: Python
+    code: |-
+      resp = client.inference.put(
+          task_type="completion",
+          inference_id="deepseek-completion",
+          inference_config={
+              "service": "deepseek",
+              "service_settings": {
+                  "api_key": "DeepSeek-Api-key",
+                  "model_id": "deepseek-chat"
+              }
+          },
+      )
+  - language: JavaScript
+    code: |-
+      const response = await client.inference.put({
+        task_type: "completion",
+        inference_id: "deepseek-completion",
+        inference_config: {
+          service: "deepseek",
+          service_settings: {
+            api_key: "DeepSeek-Api-key",
+            model_id: "deepseek-chat"
+          },
+        },
+      });
+  - language: Ruby
+    code: |-
+      response = client.inference.put(
+        task_type: "completion",
+        inference_id: "deepseek-completion",
+        body: {
+          "service": "deepseek",
+          "service_settings": {
+            "api_key": "DeepSeek-Api-key",
+            "model_id": "deepseek-chat"
+          }
+        }
+      )
+  - language: PHP
+    code: |-
+      $resp = $client->inference()->put([
+          "task_type" => "completion",
+          "inference_id" => "deepseek-completion",
+          "body" => [
+              "service" => "deepseek",
+              "service_settings" => [
+                  "api_key" => "DeepSeek-Api-key",
+                  "model_id" => "deepseek-chat"
+              ],
+          ],
+      ]);
+  - language: curl
+    code:
+      'curl -X PUT -H "Authorization: ApiKey $ELASTIC_API_KEY" -H "Content-Type: application/json" -d
+      ''{"service":"deepseek","service_settings":{"api_key":"DeepSeek-Api-key","model_id":"deepseek-chat"
+      }'' "$ELASTICSEARCH_URL/_inference/completion/deepseek-completion"'
+  - language: Java
+    code: >
+      client.inference().put(p -> p
+          .inferenceId("deepseek-completion")
+          .taskType(TaskType.Completion)
+          .inferenceConfig(i -> i
+              .service("deepseek")
+              .serviceSettings(JsonData.fromJson("{\"api_key\":\"DeepSeek-Api-key\",\"model_id\":\"deepseek-chat\"}"))
+          )
+      );
```

Comment:
Wow, did you write the alternatives yourself? This isn't needed, they are generated automatically (and checked against real clients)